src/account/getAccountDetails.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosResponse } from 'axios';
+import { AccountDetailsI } from './accountDetailsI'
+import dotenv from 'dotenv';
+dotenv.config();
+
+const baseUrl = process.env.BASE_URL;
+const authToken = process.env.AUTH_TOKEN;
+const accountId = process.env.ACCOUNT_ID;
+
+export async function httpRequest<T>(
+    method: 'GET' | 'POST' | 'PUT' | 'DELETE',
+    url: string
+): Promise<AxiosResponse<T>> {
+    const options = {
+        method,
+        headers: {
+            accept: 'application/json',
+            Authorization: `Bearer ${authToken}`,
+        },
+    };
+
+    try {
+        const response = await axios(url, options);
+        return response;
+    } catch (error) {
+        console.error(`Error making ${method} request to ${url}:`, error);
+        throw error;
+    }
+}
+
+export async function getAccountDetails(): Promise<AxiosResponse<AccountDetailsI>> {
+    const url = `${baseUrl}account/${accountId}`;
```

Comment:
what if I have to get details of other accounts, too? How are you going to handle account ids, then?

Reply:
Where do you get other account IDs? I'm calling my account details with my ID.